storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -204,16 +204,16 @@ func getAuthCredential(config *rest.Config) *salt.Credential {
 	if config.BearerToken != "" {
```

Comment:
Commit message:

```diff
- closest thing available: at least it will catch typo at compile time…
+ closest thing available: at least it will catch typos at compile time…
```

Reply:
Done